src/Compatibility/Maps/src/iOS/FormsMaps.cs

```diff
@@ -14,7 +14,7 @@ public static void Init()
 		{
 			if (s_isInitialized)
 				return;
-			GeocoderBackend.Register();
+			//GeocoderBackend.Register();
```

Comment:
Intentionally left commented?

Reply:
yes it was removed, but i can delete this

Reply:
These projecs will go away soon after merging maps. 

I will make another PR for that.